scripts/.storybook/preview.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import type { Preview } from '@storybook/react-vite';
+import { themes } from 'storybook/theming';
+import { GlobalStyles } from '@contentful/f36-core';
+
+const preview: Preview = {
+  decorators: [
+    (Story) => {
+      return (
+        <>
+          <GlobalStyles />
+          <Story />
+        </>
+      );
+    },
+  ],
+  parameters: {
```

Comment:
Is removing the [order in the sidebar](https://github.com/contentful/forma-36/blob/main/scripts/.storybook/preview.js#L19) intentional?